.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
Instructions are better as comments:
```suggestion
<!-- A clear and concise description of what the bug is. -->
```

That way, people don't have to clear them manually.

The other places as well.

Reply:
I just noticed that the other places remained.